islandora_web_archive.drush.inc

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/**
+ * @file
+ * Implementation of Drush hooks.
+ */
+
+/**
+ * Implements hook_drush_command().
+ */
+function islandora_web_archive_drush_command() {
+  return array(
+    'islandora_web_archive_migrate_png_to_screenshot' => array(
+      'bootstrap' => DRUSH_BOOTSTRAP_MAX,
+      'drupal_dependencies' => array(
+        'islandora',
+      ),
+      'description' => 'Iterates each web archive object and copies the PNG datastream into a SCREENSHOT datastream.',
+      'examples' => array(
+        'drush -u 1 islandora_web_archive_migrate_png_to_screenshot',
+      ),
+    ),
+  );
+}
+
+/**
+ * Implements hook_drush_command().
+ */
+function drush_islandora_web_archive_migrate_png_to_screenshot() {
+  // Get all islandora web archive objects.
+  $query = <<<EOQ
+    PREFIX fedora-model: <info:fedora/fedora-system:def/model#>
+    SELECT DISTINCT ?pid
+    FROM <#ri>
+    WHERE {
+      ?pid <fedora-model:hasModel> <info:fedora/islandora:sp_web_archive>
```

Comment:
Could also modify this query using the `<info:fedora/fedora-system:def/view#disseminates>` predicate to find all objects that currently have a PNG datastream as opposed to loading it and such below.